deepchem/models/torch_models/layers.py

```diff
@@ -3186,6 +3186,95 @@ def forward(
         return adjacency_tensor, node_tensor, output
 
 
+class MolGANAggregationLayer(nn.Module):
+    """
+    Graph Aggregation layer used in MolGAN model.
+    MolGAN is a WGAN type model for generation of small molecules.
+    Performs aggregation on tensor resulting from convolution layers.
+    Given its simple nature it might be removed in future and moved to
+    MolGANEncoderLayer.
+
+
+    Example
```

Comment:
This should be Examples I think https://numpydoc.readthedocs.io/en/latest/format.html#examples

Reply:
fixed